android/demos/memory-leaks/src/main/java/com/uber/rib/SampleApplication.java

```diff
@@ -31,24 +29,17 @@ public class SampleApplication extends Application implements HasActivityDelegat
   public void onCreate() {
     activityDelegate = new SampleActivityDelegate();
     super.onCreate();
-    if (!LeakCanary.isInAnalyzerProcess(this)) {
```

Comment:
Was the separate process deprecated?

Reply:
It's optional now. Most apps shouldn't need it as LeakCanary uses a lot less memory, and this complicates the setup.